src/oskari.dom.js

```diff
@@ -113,6 +113,21 @@ const isEmbedded = () => {
     return getMapContainerEl().classList.contains(APP_EMBEDDED_CLASS);
 };
 
+const showNavigation = (show) => {
+    const nav = getNavigationEl();
```

Comment:
getNavigationEl() returns null/undefined if the element isn't there so we'll need to handle that.